qiita_db/support_files/patches/python_patches/30.py

```diff
@@ -0,0 +1,78 @@
+from qiita_db.sql_connection import TRN
+from qiita_db.metadata_template.constants import (NA_VALUES, TRUE_VALUES,
+                                                  FALSE_VALUES)
+from qiita_db.metadata_template import SampleTemplate, PrepTemplate
+from qiita_db.study import Study
+
+bool_vals = set(TRUE_VALUES + FALSE_VALUES + [None])
+na_vals = set(NA_VALUES)
+
+nans = tuple(NA_VALUES)
+false_vals = tuple(FALSE_VALUES)
+true_vals = tuple(TRUE_VALUES)
+
+st_update = set()
+pr_update = set()
+
+with TRN:
+    sql = """SELECT table_name
+             FROM information_schema.tables
+             WHERE table_schema='qiita'
+                AND (table_name SIMILAR TO 'sample\_[0-9]+'
```

Comment:
I don't think so, but this works fine regardless because it makes sure that only integers follow the underscore, so the 0 is needed in the brackets.